roofit/hs3/test/testRooFitHS3.cxx

```diff
@@ -44,9 +44,57 @@ TEST(RooFitHS3, RooArgusBG)
    RooAddPdf model("model", "g+a", RooArgList(signalModel, background), RooArgList(nsig, nbkg));
 
    std::string rootetcPath = gSystem->Getenv("ROOTSYS");
-   RooJSONFactoryWSTool::loadExportKeys(rootetcPath + "/etc/root/RooFitHS3_wsexportkeys.json");
+   RooJSONFactoryWSTool::loadExportKeys(rootetcPath + "/etc/RooFitHS3_wsexportkeys.json");
    RooWorkspace work;
    work.import(model);
    RooJSONFactoryWSTool tool(work);
    tool.exportJSON("argus.json");
 };
+
+#include "RooRealVar.h"
+#include "RooSimultaneous.h"
+#include "RooProdPdf.h"
+#include "RooCategory.h"
```

Comment:
Please move all include statements to the top of the file (and while you are at it, the `-#ifndef __CINT__` guard at the top of the file should be removed).